web/src/main/java/org/akaza/openclinica/service/ImportServiceImpl.java

```diff
@@ -104,7 +113,17 @@
 
     @Autowired
     OpenRosaSubmissionController openRosaSubmissionController;
+    
+    @Autowired
+    @Qualifier( "dataSource" )
+    private BasicDataSource dataSource;
```

Comment:
Can you please remove this identifier from here ,I don't think we are using it anywhere in this file.